<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>339</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>616</width>
    <height>339</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>616</width>
    <height>354</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pokedex</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/Icones/ressources/pokeball.png</normaloff>:/Icones/ressources/pokeball.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QListWidget" name="pokemonList">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>298</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>298</height>
         </size>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="whatsThis">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Liste des pok√©mons&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="currentRow">
         <number>-1</number>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="pokemonIcon">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Icon</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonName">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="pokemonDescription">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="text">
             <string>Description</string>
            </property>
            <property name="textFormat">
             <enum>Qt::PlainText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>#ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonId">
            <property name="text">
             <string>#ID</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonType">
            <property name="text">
             <string>Type</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ability</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonAbility">
            <property name="text">
             <string>Ability</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Exp.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonExp">
            <property name="text">
             <string>Exp.</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Ratio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonRatio">
            <property name="text">
             <string>Ratio</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_9">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Egg Group</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonEggGroup">
            <property name="text">
             <string>Egg Group</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Species</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonSpecies">
            <property name="text">
             <string>Species</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonHeight">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label_15">
            <property name="maximumSize">
             <size>
              <width>60</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Weight</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="pokemonWeight">
            <property name="text">
             <string>Weight</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Moves</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="moveList">
          <property name="minimumSize">
           <size>
            <width>209</width>
            <height>150</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>209</width>
            <height>150</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Evolutions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="evoList">
          <property name="minimumSize">
           <size>
            <width>209</width>
            <height>90</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>209</width>
            <height>90</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Nouvel √©l√©ment</string>
           </property>
           <property name="icon">
            <iconset resource="../ressources.qrc">
             <normaloff>:/Icones/ressources/creer.png</normaloff>:/Icones/ressources/creer.png</iconset>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>616</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEquipes">
    <property name="title">
     <string>Equipes</string>
    </property>
    <addaction name="actionNouvelleEquipe"/>
    <addaction name="actionGererEquipes"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAide">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="actionDocumentation"/>
    <addaction name="separator"/>
    <addaction name="actionApropos"/>
    <addaction name="actionAproposQT"/>
   </widget>
   <widget class="QMenu" name="menuPokedex">
    <property name="title">
     <string>Pokedex</string>
    </property>
    <addaction name="actionFermer"/>
   </widget>
   <widget class="QMenu" name="menuDresseurs">
    <property name="title">
     <string>Dresseurs</string>
    </property>
    <addaction name="actionNouveauDresseur"/>
    <addaction name="actionGererDresseurs"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuPokedex"/>
   <addaction name="menuDresseurs"/>
   <addaction name="menuEquipes"/>
   <addaction name="menuAide"/>
  </widget>
  <action name="actionNouvelleEquipe">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/creer.png</normaloff>:/Icones/ressources/creer.png</iconset>
   </property>
   <property name="text">
    <string>Nouvelle equipe</string>
   </property>
  </action>
  <action name="actionGererEquipes">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/gerer.png</normaloff>:/Icones/ressources/gerer.png</iconset>
   </property>
   <property name="text">
    <string>Gerer les equipes</string>
   </property>
  </action>
  <action name="actionSauvegarder">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/enregistrer.png</normaloff>:/Icones/ressources/enregistrer.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionEffacer">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/fermer.png</normaloff>:/Icones/ressources/fermer.png</iconset>
   </property>
   <property name="text">
    <string>Tout effacer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionDocumentation">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/aide.png</normaloff>:/Icones/ressources/aide.png</iconset>
   </property>
   <property name="text">
    <string>Documentation</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionApropos">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/apropos.png</normaloff>:/Icones/ressources/apropos.png</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
  <action name="actionFermer">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/fermer.png</normaloff>:/Icones/ressources/fermer.png</iconset>
   </property>
   <property name="text">
    <string>Fermer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="actionAproposQT">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/cute.png</normaloff>:/Icones/ressources/cute.png</iconset>
   </property>
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionNouveauDresseur">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/creer.png</normaloff>:/Icones/ressources/creer.png</iconset>
   </property>
   <property name="text">
    <string>Nouveau dresseur</string>
   </property>
  </action>
  <action name="actionGererDresseurs">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/gerer.png</normaloff>:/Icones/ressources/gerer.png</iconset>
   </property>
   <property name="text">
    <string>Gerer les dresseurs</string>
   </property>
  </action>
  <action name="actionSauvegarderD">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/enregistrer.png</normaloff>:/Icones/ressources/enregistrer.png</iconset>
   </property>
   <property name="text">
    <string>Sauvegarder</string>
   </property>
  </action>
  <action name="actionEffacerD">
   <property name="icon">
    <iconset resource="../ressources.qrc">
     <normaloff>:/Icones/ressources/fermer.png</normaloff>:/Icones/ressources/fermer.png</iconset>
   </property>
   <property name="text">
    <string>Tout effacer</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pokemonList</tabstop>
 </tabstops>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
